app/views/computacenter/home/show.html.erb

```diff
@@ -38,7 +38,13 @@
       <%= govuk_link_to 'Responsible bodies managing multiple Chromebook domains', computacenter_multi_domain_chromebooks_path %>
     </h2>
 
-    <p class="govuk-body">View details of responsible bodies that manage schools with mutliple Chromebook domains.</p>
+    <p class="govuk-body">View details of responsible bodies that manage schools with multiple Chromebook domains.</p>
+
+    <h2 class="govuk-heading-m">
+      <%= govuk_link_to 'ISS or Social Care LAs managing multiple Chromebook domains', computacenter_multi_domain_chromebooks_iss_scl_path %>
```

Comment:
Did you mean to use the new `multi_domain_chromebooks_iss_scl` i18n key here?

Reply:
It should but nothing of that entire page is using it. It is however used on the title of the page it links to which all the other pages do, so I'm tempted to follow the "bad" consistency for now, it's a coincidence it's the "same" string but it could be different.

Reply:
Ahh, gotcha! Didn't realise that the new key was under the `page_titles` "namespace" – all good.